### ISSUE TITLE
Northeastern cave entrance displays some surface Z-level tiles in the cavern and vice-versa

### ISSUE THREAD

**Issue by Serithi** [PolarisSS13/Polaris#8498]
#### Brief description of the issue
What it says on the tin.

#### What you expected to happen
Not that.

#### What actually happened
That.

#### Steps to reproduce
Go to the surface-level cavern entrance to the northeast (and check any other such entrances) and take a look.

#### Additional info:
 - **Server Revision**: master - 2022-03-30 21ee12ae4b12d499d805dc6bfda751fd57cc957a
 
 Around the corner in the cavern.
 
![1](https://user-images.githubusercontent.com/1519721/161174394-6cb19118-54eb-4d43-b49e-088a0e909d63.png)

By the surface entrance.

![2](https://user-images.githubusercontent.com/1519721/161174428-ef41c93b-a877-4c5e-b5a1-a43f362a1604.png)

Around the other corner on the surface.

![3](https://user-images.githubusercontent.com/1519721/161174447-a627418c-d981-42b3-a677-dc9c4e38588d.png)

**Comment by PrismaticGynoid** [comment]
Yeah, 514 broke portal rendering somehow. You shouldn't be able to see _through_ a portal without being able to _see_ the portal, but something about that changed with the update, and now it's visible through blocks.
